src/pages/TabHome.tsx

```diff
@@ -75,6 +75,135 @@ const InfoButton = (props: InfoButtonProps) => {
   );
 };
 
+interface SelectCalendarProps {
+  setIsEditCalendar: (newIsOpen: boolean) => void;
+}
+
+const ViewCalendar = (props: SelectCalendarProps) => {
+  const { t } = useTranslation();
+  const { cycles } = useContext(CyclesContext);
+
+  const lastPeriodDays = getLastPeriodDays(cycles);
+
+  return (
+    <IonDatetime
+      style={{ borderRadius: "20px" }}
+      color="light"
+      presentation="date"
+      locale={getCurrentTranslation()}
+      size="cover"
+      firstDayOfWeek={1}
+      highlightedDates={(isoDateString) => {
+        if (cycles.length === 0) {
+          return undefined;
```

Comment:
Look like the same as `return`

Reply:
`return undefined` is written in the documentation